mautic/mautic
# Disable trackable for SMS in configuration

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:
Disable tracking URLs for SMS is now related to Twilio. This PR move it to configuration. Then it's not related just to one integration. 




#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2.  Check Twilio plugin If there is no option Disable trackable urls like before
3. Go to Configuration - Test messsages -  see new option Disable trackable urls
4. Enable it
5. Send SMS with link
6. Check If SMS was not send with trackable URLs
